snapshot-labs/sx-monorepo
# chore: connect services when running dev:full

### Summary

Now when running dev:full UI will automatically use local sx-api and mana.

### How to test

1. Set up `.env` in `apps/mana` and `apps/api` (you can copy those files from your previous projects, but make sure `DATABASE_URL` is not changed from `.env.example`.
2. Run `yarn dev:full`.
3. Open UI, it uses local services (for Starknet sepolia and Mana).
